content/en/tracing/profiling/_index.md

```diff
@@ -110,14 +120,13 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 
 | Environment variable                             | Type          | Description                                                                                      |
 | ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
-| `DD_API_KEY`                                     | String        | The [Datadog API key][1] to use when uploading profiles. New in version 0.37.                                        |
+| `DD_API_KEY`                                     | String        | The [Datadog API key][1] to use when uploading profiles.                                        |
```

Comment:
Why did you remove these? There was discussion that went into adding them, including looking at how many popular open source work, and thinking about what would best serve users of various ~personas.

Reply:
Oh sorry, I wasn't aware of that! I'll add it back :)